mon-pix/app/controllers/certifications/start.js

```diff
@@ -0,0 +1,12 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+
+export default Controller.extend({
+  currentUser: service(),
+  showCongratulationsBanner: true,
+  actions: {
+    closeBanner() {
+      this.set('showCongratulationsBanner', false);
```

Comment:
tu peux utiliser `this.toggleProperty(showCongratulationsBanner);` :)

Reply:
Vu le nom de la fonction, je préfère un set false ici

Reply:
Je serai étonné d'appeler deux fois une fonction qui s'appelle `closeUnTruc()` et que la deuxième fois ça fasse ré-aparaître ledit truc